### ISSUE TITLE
This is in response to the response I recieved in the issue #12228. Please listen to what I have to say... as you are not letting me comment on that issue.

### ISSUE THREAD

**Issue by ghost** [mozilla-mobile/fenix#12380]
Developers, do you have a plan to migrate users and their data  to the "Release" version who might have installed the "Firefox Beta" version mistakenly by assuming it was a "Release version" from the experiment you did: #11165? Do you plan to notify them that they were a part of an experiment?

And do you have a plan to migrate the user data of the users to the current "Firefox Beta" version from the "Firefox Nightly" which was "Firefox Preview" a few days ago... (I am hoping I am explaining this clearly)?
And further after that to the release version when Fenix is released? 

Since both were "Release"/"Production" versions. 

There are some users that who don't want to sign up for Firefox Account to migrate their data even after explaining that it is End-to-End encrypted and they can delete the Firefox Account after they migrate their data if they want and even if they did they wouldn't have all their data migrated since Sync is not able to sync all data yet (I know there are considerations to add some data sync types like Top Sites, Collections, etc.), but I think a local data migration/backup tool is necessary for those who do not want to create Firefox Accounts. I received a [response](https://github.com/mozilla-mobile/fenix/issues/12228#issuecomment-655258324) on the issue I created: #12228. But it doesn't talk about taking any steps to resolve the issues I am raising, I was planing to add these concerns to that issue itself but before I could do so you locked that thread, this is the reason I had to create another issue to ask further questions. I was never going to create the issue #12228 and was gonna respond to the developers in the issue https://github.com/mozilla-mobile/fenix/issues/12207 itself but before I could do so, you locked that thread as "off-topic" as well. In fact I was gonna thank them for responding but you didn't even let me do that as well. 

I am not being arrogant or want any attention, In fact I do not want anything but to help improve 
Firefox (because I am in love with what you are building and doing for the open web!). The type of [reactions and comments](https://github.com/mozilla-mobile/fenix/issues/12228#issuecomment-653538335) I received made me really think that if I was overreacting and my concerns were overblown. Do you also think that this is the case? if so, I will stop criticizing right now! Since sometimes I tend to overthink some matters. :grinning: 

I was not what you call "ranting" in the issue #12228 as some users might have thought! I thought my concerns were real, that is the reason why I filed the issue in GitHub even after I was against it (as GitHub is for filing real bugs and issues) and here I am doing it again, What a mess am I! I even tried creating a [Matrix account](https://chat.mozilla.org/#/room/%23fx-android-community:mozilla.org) to say all of this instead of creating a new GitHub issue but the amount of information overload and unfamiliarity I got there was overwhelming so I deleted the account right away. Sorry... 

For example, these were the comments I tried making before filing the issue #12228; https://github.com/mozilla-mobile/fenix/issues/12207#issuecomment-653302051, https://github.com/mozilla-mobile/fenix/issues/12207#issuecomment-653319342, https://github.com/mozilla-mobile/fenix/issues/12208#issuecomment-653364864, https://github.com/mozilla-mobile/fenix/issues/12208#issuecomment-653364864. And I never said to not have several versions of Fenix on the Play Store, they were inevitable, I know. And I can even give you my Reddit username (not already listing it because of privacy concerns, but if you really want that then I am ready to give it) to make sure to you that I have a lot patient and have always supported Mozilla and try to calm everyone down when everyone says those versions were unnecessary and and are confused and/or very angry at what Mozilla is doing;  I always tried to help and provide solutions if I can. I even created posts on Reddit which tried explaining all this situation and have always helped to redirect users on GitHub to file bug reports/feature requests/feedback to the issues they were having and have always helped them in any way I can. As I have already said I created the issues or made comments on GitHub not because _I_ was confused but because other users were having problems. I even said that I won't contribute to this repository in the comment https://github.com/mozilla-mobile/fenix/issues/12228#issuecomment-653521637 and you know what, I am not selfish enough to actually do that! but before I could say so, once again, the issue was locked. And not to mention I have lost the ability to edit, delete, or modify the comments I made; what if I want to delete some private information in them. You have taken away the choice and control from me to do that; at least you could have let me retain the control of the comments _I_ made and/or let me add further comments to the issues _I_ filed.

I will continue to help the community in whatever way I can!... because you know why -- I believe in what Mozilla believes! 

And English is not my native language and it is a language that I only learned to speak in recent years. Perhaps that is the reason why I received those type of comments from other users when I commented and created issues on GitHub as I might not have been able to convey the thoughts I was going to say clearly. I hope it is not that but it could be a real possibility in my case. :grinning: and I apologize if I made anybody upset and have hurt their feelings.

- - -
CC @vesta0 

- - -
**Edit:**
>https://github.com/mozilla-mobile/fenix/pull/12394

With the above mentioned Pull request, I think the Developers seem to have ammended the guidelines to indicate to not create issues like the ones I have filed and to not tag any of the developers team (okay, I won't do this from now on! I hardly try to tag anybody unless I think it is really important!). Please know that I already know those guidelines but if you are so un-welcoming to new users, they're not going file any issues at all. They don't even know GitHub is the place to file bug reports/feature requests and instead you can see them posting them on Reddit and other websites. Even I didn't know GitHub was the place to file issues a few months ago. And they are already afraid of creating issues thinking only the very experienced, most knowledgeable and software developers can file issues on GitHub or bug tracker! If you make those guidelines even more stricter than they already are, you'll scare them off! And most general users, if not everybody; is going to create new accounts (Matrix) and learn all those new things to say the concerns they have. 

I promise to not create any new issues like these. Have you even seen me doing this? It is only recently that I started doing this even though I knew this is not the correct way to do this as I didn't know where to file issues like this. I even thought to make a post on reddit instead of creating an issue on GitHub! 

Creating issues like these took a _huge_ toll on my emotional well-being! And not to mention the sort of comments and reactions I get after creating the issues like these! If you really want me to stop this, I'll delete my GitHub account right now or stop filing any feedback to this repository right this moment; the only reason I created GitHub account was because I wanted to help improve Firefox... 

...And I'm deleting it now. 

Thanks! :grinning:

**Comment by Oymate** [comment]
TL;DR 
Reduce channels of release(WIP)
Make a local migration solution without accounts.
Use emojis  :)
Don't us bold, italic. (it's distracting)
Chill
One problem per issue request.

**Comment by SS1113** [comment]
@Oymate 

Thanks for the "Tl;dr" for the comments I had to say. :) It means a lot to me! ;)

> One problem per issue request.

I am waiting for the Developers response and I'll open a new issue for each concern I have. I'm not willing to file issues which developers are not willing to consider!

> Reduce channels of release (WIP)

I know that already, how many times I have to tell! What I'm concerned about is what happens next; after and during the migration.


>TL;DR

And also can you explain _your_ "Tl;dr" to me? I'm not able understand it clearly? :)
 
> Use emojis :)

What do you mean by use "emojis"? I'm not good at using them. Can you explain, if I'm using it in the wrong place and a wrong time? Provide me some tips. ;)

>Don't use bold, italics. (it's distracting) 

But GitHub allows me to... ;) 

>Chill

Chilling is not what I'm good at. Hoping you could provide me some tips to "Chill". ;)


Thanks! :)

**Comment by SS1113** [comment]
@sheikh-azharuddin can you please provide your opinions on how can Mozilla make the experience better?

**Comment by SS1113** [comment]
>https://github.com/mozilla-mobile/fenix/pull/12394

With the above mentioned Pull request, think the Developers seem to have ammended the guidelines to indicate to not create issues like the ones I have filed and to not tag any of the developers team (okay, I won't do this from now on! I hardly try to tag anybody unless I think it is really important!). Please know that I already know those guidelines but if you are so un-welcoming to new users, they're not going file any issues at all. They don't even know GitHub is the place to file bug reports/feature requests and instead you can see them posting them on Reddit and other websites. Even I didn't know GitHub was the place to file issues a few months ago. And they are already afraid of creating issues thinking only the very experienced, most knowledgeable and software developers can file issues on GitHub or bug tracker! If you make those guidelines even more stricter than they already are, you'll scare them off! And most general users, if not everybody; is going to create new accounts (Matrix) and learn all those new things to say the concerns they have. 


I promise to not create any new issues like these. Have you even seen me doing this? It is only recently that I started doing this even though I knew this is not the correct way to do this as I didn't know where to file issues like this. I even thought to make a post on reddit instead of creating an issue on GitHub! 

Creating issues like these took _huge_ toll on my emotional well-being! And not to mention the sort of comments and reactions I get after creating the issues like these! If you really want me to stop this, I'll delete my GitHub account right now or stop filing any feedback to this repository right this moment; the only reason I created GitHub account was because I wanted to help improve Firefox.

Thanks! :)
